src/components/MultipleSelector.component.js

```diff
@@ -81,7 +81,6 @@ class MultipleSelector extends React.Component {
         organisationUnitsCapture: "assignToOrgUnitsCapture",
         dataViewOrganisationUnits: "assignToOrgUnitsOutput",
     };
-
```

Comment:
[very minor] I've revert this change, we typically don't commit this changes if the file was not changed for the task (it's ok to tidy up files in the general case, though)